fastlane/lanes/release.rb

```diff
@@ -157,18 +304,39 @@
   desc 'Trigger the final release build on CI'
   lane :finalize_release do |options|
     UI.user_error!('To finalize a hotfix, please use the finalize_hotfix_release lane instead') if ios_current_branch_is_hotfix
+    ensure_git_branch(branch: '^release/') # Match branch names that begin with `release/`
```

Comment:
To me, the code is already self explanatory:

```suggestion
    ensure_git_branch(branch: '^release/')
```